js/project-list.js

```diff
@@ -0,0 +1,58 @@
+/* Project Card */
+
+var ProjectList = {
+  template: _.template($("script#project-list-template").html()),
+  buildHTML: function(projectData) {
+    var id = projectData.id;
+    var timestamp = new Date(projectData.Timestamp);
+    var month = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"][timestamp.getMonth()];
+    var issues = projectData.Issues ? projectData.Issues : "";
+    var issuesAsArray = issues.split(", ");
+    var dataForTemplate = {
```

Comment:
Yeah I thought that as well but I'm not very sure for some of the meta data. In case I don't remove some unnecessary data or by mistake remove required data, kindly tell me or do it for me.